seeds/data/projects.json

```diff
@@ -300,5 +300,86 @@
     "licenceNumber": "PR-X00005",
     "licenceHolderId": "5b7bad13-f34b-4959-bd08-c6067ae2fcdd",
     "status": "inactive"
+  },
+  {
+    "id": "b8973df0-408c-4f3c-a7f2-b5b8f8c2943b",
+    "title": "RA true",
+    "schemaVersion": 1,
+    "status": "active",
```

Comment:
I think setting active projects without a grant date or expiry date might cause some weirdness down the road.

Reply:
Looks like this was resolved